Program.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
```

Comment:
Most of this imports aren't actually needed/required. VSCode tells me we only actually need:

```
using Microsoft.AspNetCore.Hosting;
using Microsoft.Extensions.Hosting;
```

Reply:
I removed the unused imports.